packages/bundle-source/src/main.js

```diff
@@ -13,6 +13,16 @@ const USAGE =
  * @returns {Promise<void>}
  */
 export const main = async (args, { loadModule, pid, log }) => {
+  let noTransforms = false;
```

Comment:
suggestion: if we're assuming Node.js, [util.parseArgs](https://nodejs.org/api/util.html#utilparseargsconfig) was created for the "just accept a couple options" use-case

Reply:
Done.